lib/default.nix

```diff
@@ -90,7 +90,7 @@ let
       concatImapStringsSep makeSearchPath makeSearchPathOutput
       makeLibraryPath makeBinPath optionalString
       hasInfix hasPrefix hasSuffix stringToCharacters stringAsChars escape
-      escapeShellArg escapeShellArgs replaceChars lowerChars
+      escapeShellArg escapeShellArgs escapeRegex replaceChars lowerChars
```

Comment:
I don't like the pattern of exposing all functions under `lib.*` when they are also namespaced under `lib.strings.*`. So I think it's best to not continue with this by not adding `escapeRegex` here.